src/token/mod.rs

```diff
@@ -166,6 +166,24 @@ where
 /// nested pipelines (the nesting must be done indirectly through an init-file),
 /// second that Rust Geodesy does not support init-files. Hence no support for
 /// any kind of nesting here.
+///
+/// ## Known differences between PROJ and Rust Geodesy definitions:
+///
+/// ## Ellipsoid definitions
+/// - Geodesy only supports a limited set of builtin ellipsoids OR or definition
+/// via semi-major and reverse-flattening parameters  `ellps=a,rf`.
+/// - PROJ has [richer ellipsoid](https://proj.org/en/9.3/usage/ellipsoids.html#ellipsoid-size-parameters)
+/// support which *parse_proj* provides partial support for.
+/// - Specifically if an ellipsoid is defined via `a` and `rf` parameters, *parse_proj*
+/// will redefine them as `ellps=a,rf` and remove the `a` and `rf` parameters.
+/// - All other cases supported by PROJ are NOT handled by *parse_proj* and will
+/// fail when instantiating the operator.
+///
+/// ## Scaling via `k` parameter
+/// - PROJ still supports the deprecated `k` parameter. Most output from `projinfo` will
+/// have the scaling defined as `k` instead of `k_0`.
+/// - *parse_proj* will replace `k` with `k_0` when ever it is encountered.
```

Comment:
[whenever](https://www.dailywritingtips.com/whenever-vs-when-ever/)

Reply:
Ah yes than you, that is a useful link.